src/Tizen.Applications.Common/Tizen.Applications.RPCPort/Parcel.cs

```diff
@@ -181,6 +182,37 @@ public Parcel(Port port)
                 throw new InvalidIOException();
         }
 
+        /// <summary>
+        /// Constructor with the raw bytes.
+        /// </summary>
+        /// <param name="bytes">The raw bytes.</param>
+        /// <exception cref="InvalidIOException">Thrown when an internal IO error occurs.</exception>
+        /// <since_tizen> 9 </since_tizen>
+        public Parcel(byte[] bytes)
+        {
+            if (bytes == null)
+                throw new InvalidIOException();
+            var r = Interop.LibRPCPort.Parcel.CreateFromRaw(out _handle, bytes, (uint)bytes.Length);
+            if (r != Interop.LibRPCPort.ErrorCode.None)
+                throw new InvalidIOException();
+        }
+
+        /// <summary>
+        /// Gets the raw bytes of the parcel.
+        /// </summary>
+        /// <returns>The raw bytes of the parcel.</returns>
+        /// <exception cref="InvalidIOException">Thrown when an internal IO error occurs.</exception>
+        /// <since_tizen> 9 </since_tizen>
+        public byte[] Marshall()
```

Comment:
`Marshal`은 unmanaged 데이터와 managed데이터간 변환을 하는 의미라서 여기서의 Marshal을 사용하는것은 적합하지 않은것 같습니다.

일반적으로 형태 변경에 사용되는 네이밍을 사용하시는것을 추천드립니다.
```suggestion
        public byte[] ToBytes()
```

Reply:
수정하였습니다. 감사합니다.